modules/backend/widgets/Filter.php

```diff
@@ -789,6 +856,46 @@ protected function datesFromAjax($ajaxDates)
     }
 
     /**
+     * Convert an array from the posted numbers
+     *
+     * @param  mixed $scope
+     * @param  array $numbers
+     *
+     * @return array
+     */
+    protected function numbersFromAjax($ajaxNumbers)
+    {
+        $numbers = [];
+        $dateRegex = '/\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}/';
+
+        if (null !== $ajaxNumbers) {
+            if (!is_array($ajaxNumbers)) {
+        //        if(preg_match($dateRegex, $ajaxDates)) {
```

Comment:
Commented out code is not acceptable unless it is part of the inline documentation.